content/bn/docs/reference/glossary/cgroup.md

```diff
@@ -0,0 +1,17 @@
+---
+title: সিগ্রুপ (কন্ট্রোল গ্রুপ)
+id: cgroup
+date: 2019-06-25
+full_link:
+short_description: >
+  লিনাক্স প্রসেসের একটি গ্রুপ যা ঐচ্ছিক সম্পদের আলাদাীকরণ, হিসাব এবং সীমার সাথে সংযুক্ত।
```

Comment:
@Noblesse013 , Thanks for the contribution.

Here are some recommendations that we can use in terms of localization. 

```
অপশনাল রিসোর্স আইসোলেশন, একাউন্টিং এবং লিমিট সহ লিনাক্স প্রসেসেগুলির একটি গ্রুপ।
```